src/hooks/useUserBalance.ts

```diff
@@ -0,0 +1,70 @@
+import { getColonyNetworkClient, Network } from '@colony/colony-js';
```

Comment:
This is temporary. I expect this info to be accessible in the db once https://github.com/JoinColony/colonyCDapp/issues/286 has been finished